upf/upf_cli.c

```diff
@@ -1646,6 +1646,67 @@ VLIB_CLI_COMMAND (upf_add_policy_command, static) =
 };
 /* *INDENT-ON* */
 
+static clib_error_t *
+upf_pfcp_heartbeats_command_fn (vlib_main_t * vm,
+				unformat_input_t * main_input,
+				vlib_cli_command_t * cmd)
+{
+  unformat_input_t _line_input, *line_input = &_line_input;
+  upf_main_t *gtm = &upf_main;
+  clib_error_t *error = NULL;
+  u32 timeout, retries = 0;
+  int rv = 0;
+
+  if (!unformat_user (main_input, unformat_line_input, line_input))
+    return 0;
+
+  while (unformat_check_input (line_input) != UNFORMAT_END_OF_INPUT)
+    {
+      if (unformat (line_input, "timeout %u", &timeout))
+	;
+      else if (unformat (line_input, "retries %u", &retries));
+      else
+	return (clib_error_return (0, "unknown input '%U'",
+				   format_unformat_error, line_input));
+    }
+
+  rv = vnet_upf_pfcp_heartbeat_config (timeout, retries);
+  if (rv)
+    error =
+      clib_retur_error
```

Comment:
Typo: `clib_retur_error`. This will fail during runtime with an undefined symbol error